README.md

```diff
@@ -259,6 +259,21 @@ Examples:
 
 Additional details are available from [`ssh_config(5)`](https://linux.die.net/man/5/ssh_config)
 
+#### SSH_POST_SCRIPT
+
+When set, SSH_POST_SCRIPT will be the location of a shell script to execute after starting autossh.
+This is useful to send some details to the server about the autossh client.
+For example, if a random port is used, then it's useful for the server to know who is on this port.
```

Comment:
> it's useful for the server to know who is on this port.

This does not make sense just reading it.

Do you mean to say you can associate a specific container/instance of autossh is using a specific port remotely?  I'm a bit confused as to the usecase.